### DISCUSSION TITLE
Ability to change order of buffer tabs

### DISCUSSION THREAD

**Discussion by nDimensionalSpace** [NvChad/NvChad#2180] [Q&A] [answered]
Is this possible?  I see the keymaps to change the view among buffers, but not how to change the order of the buffer tabs themselves.  Thanks.

**Comment by siduck** [comment, marked as answer]
search through our ui repo issues or check move_buf function in tabufline/init.lua of ui repo

**Reply by nDimensionalSpace** [reply]
@siduck Heh, I had seen some references to move_buf around, but couldn't find it in any of the files in the main or example_config repos . . .  Thanks.

**Reply by siduck** [reply]
@nDimensionalSpace please check ui repo's tabufline dir's init.lua :(
